nautobot/dcim/templates/dcim/rackreservation_edit.html

```diff
@@ -1,24 +1,2 @@
 {% extends 'generic/object_create.html' %}
-{% load form_helpers %}
-
-{% block form %}
-    <div class="panel panel-default">
-        <div class="panel-heading"><strong>Rack Reservation</strong></div>
-        <div class="panel-body">
-            {% render_field form.location %}
-            {% render_field form.rack_group %}
-            {% render_field form.rack %}
-            {% render_field form.units %}
-            {% render_field form.user %}
-            {% render_field form.description %}
-        </div>
-    </div>
-    <div class="panel panel-default">
-        <div class="panel-heading"><strong>Tenant Assignment</strong></div>
-        <div class="panel-body">
-            {% render_field form.tenant_group %}
-            {% render_field form.tenant %}
-        </div>
-    </div>
```

Comment:
Do we still have this logical grouping and ordering of the form fields after removing this? Otherwise this seems over-aggressive removal of code.